src/Toolchain/ToolchainEnv.ts

```diff
@@ -33,7 +33,7 @@ import {showPasswordQuickInput} from '../View/PasswordQuickInput';
 class Env implements BuilderJob {
   logTag = 'Env';
   workFlow: WorkFlow;  // our build WorkFlow
-  currentWorkspace: string = '';
+  currentWorkspace: string = require('os').homedir();
```

Comment:
Since `workspace` is defined in vscode, it can make confusion to others. Let's change this variable name to `workingDirectory`